src/logic/currencyValues/api/fetchAvailableCurrencies.ts

```diff
@@ -1,6 +1,11 @@
-import { getFiatCurrencies, GatewayDefinitions } from '@gnosis.pm/safe-react-gateway-sdk'
-import { getClientGatewayUrl } from 'src/config'
+import axios from 'axios'
 
-export const fetchAvailableCurrencies = async (): Promise<GatewayDefinitions['FiatCurrencies']> => {
-  return getFiatCurrencies(getClientGatewayUrl())
+export const fetchAvailableCurrencies = async (): Promise<string[]> => {
+  const geckoCurrencies = (
+    await axios.get<string[]>('https://api.coingecko.com/api/v3/simple/supported_vs_currencies')
+  ).data.map((currency) => currency.toUpperCase())
+  const fiatCurrencies = (
+    await axios.get<{ [currency: string]: string }>('https://openexchangerates.org/api/currencies.json')
+  ).data
```

Comment:
is this doing fetches in a map? 

Suggest a Promise.all for parallel fetching

Reply:
This happens just one time on load so I didn't think it was worth it

Reply:
not worth it. is there a cost?